Bio/PDB/DSSP.py

```diff
@@ -412,16 +412,21 @@ def __init__(
             Sander & Rost (1994), or Wilke: Tien et al. 2013, as string
             Sander/Wilke/Miller. Defaults to Sander.
         file_type: string
-            File type switch, either PDB or DSSP with PDB as default.
+            File type switch: either PDB, MMCIF or DSSP. Inferred from the
+            file extension by default.
 
         """
         self.residue_max_acc = residue_max_acc[acc_array]
 
         # create DSSP dictionary
+        if file_type == "":
+            file_type = os.path.splitext(in_file)[1][1:]
         file_type = file_type.upper()
-        assert file_type in ["PDB", "DSSP"]
-        # If the input file is a PDB file run DSSP and parse output:
-        if file_type == "PDB":
+        if file_type == "CIF":
+            file_type = "MMCIF"
+        assert file_type in ["PDB", "MMCIF", "DSSP"]
```

Comment:
I would add an error message that makes it clear to the user what the issue is, otherwise they'll get an `AssertionError` that might not be so explicit.

Reply:
Good idea, fixed it. Thanks for the review.